docs(v4): update changesets (#3688)
refinedev/refine: .changeset/slow-wasps-remain.md

"@pankod/refine-core": minor
---

> -   `congig` prop of `useList` and `useInfiniteList` hooks are deprecated. Use `sorters`, `filters`, `pagination` and `hasPagination` props instead.

Reviewer: ```suggestion
-   `useList` and `useInfiniteList`'s `congig` prop is now deprecated. Use `sorters`, `filters`, `pagination` and `hasPagination` props instead.
```
